lib/cmd/common/env.js

```diff
@@ -131,37 +113,32 @@ function listDeployed(appId, envValue, includeSys, cb) {
   common.listAppEnv(appId, envValue, function (err, currentEnvs) {
     common.listDeployedAppEnv(appId, envValue, function (erro, deployedEnvs) {
       var allEnvs = {};
-      if (currentEnvs.list) {
-        for (var i = 0; i < currentEnvs.list.length; i++) {
-          var fields = currentEnvs.list[i].fields;
-          if (allEnvs[fields.name]) {
-            allEnvs[fields.name].current = fields[targetField];
-          } else {
-            allEnvs[fields.name] = {current: fields[targetField]};
-          }
+
+      _.each(currentEnvs.list, function (env) {
+        var fields = env.fields;
+        if (!allEnvs[fields.name]) {
+          allEnvs[fields.name] = {}
```

Comment:
![BLOCKER](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-blocker.png) Add a semicolon at the end of this statement. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://metrics.skunkhenry.com/coding_rules#rule_key=javascript%3ASemicolon)